website_menu_url_translate/controllers/main.py

```diff
@@ -0,0 +1,56 @@
+import werkzeug.utils
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.web.controllers.main import Home
+
+
+class Website(Home):
+    @http.route(
+        "/website/lang/<lang>",
+        type="http",
+        auth="public",
+        website=True,
+        multilang=False,
+    )
+    def change_lang(self, lang, r="/", **kwargs):
```

Comment:
Is it possible to not completely override the super?

Reply:
@tarteo, Yes, I've added the fix that is mentioned above and did not overwrite the change_lang function.
Could you please review it?

Thank you